solutions/observability/logs/logs-index-template-defaults.md

```diff
@@ -0,0 +1,76 @@
+---
+navigation_title: Default `logs` index template
+description: Learn what the default component templates and ingest pipeline hooks for logs do.
+applies_to:
+  stack: all
+  serverless: all
+products:
+  - id: observability
+---
+
+# Default `logs` index template
+
+Elastic applies a managed index template to data streams that follow the `logs-*-*` index pattern. This index template references a set of reusable component templates and an optional `@custom` ingest pipeline that standardizes how log data streams are mapped, indexed, and preprocessed.
+
+The managed `logs` index template has a priority of `100` and is automatically applied to any data stream following the `logs-*-*` index pattern, unless you override it with your own higher-priority index template.
+
+Depending on your deployment, the `logs` index template applies one of the following data retention policies by default:
+
+* In {{stack}} (self-managed and {{ecloud}}): lifecycle is managed by [Index Lifecycle Management (ILM)](../../../manage-data/lifecycle/index-lifecycle-management.md). By default, rollover occurs when the primary shard reaches 50 GB or the index age reaches 30 days.
+* In {{serverless-full}}: lifecycle is managed by [Data Stream Lifecycle (DSL)](../../../manage-data/lifecycle/data-stream.md). By default, logs are retained for 30 days. 
+
+Refer to [Logs index template reference](logs-index-template-reference.md) for instructions on how to view or edit the logs index template in {{kib}}.
+
+## Component templates
+
+The managed `logs` index template is composed of the following component templates:
+
+### `logs@mappings`
+
+Provides general mappings for logs data streams:
+
+* Disables automatic date detection for string fields to avoid mis-parsing.  
+* Defines ECS `data_stream.*` fields:
+  * `data_stream.type`: constant_keyword, value `logs`  
+  * `data_stream.dataset`: constant_keyword, for example `nginx.access` (must be ≤ 100 characters, no `-`)
+  * `data_stream.namespace`: constant_keyword, for example `production` (must be ≤ 100 characters, no `-`)
+
+### `logs@settings`
+
+Configures default index settings for logs data streams: 
+
+  * Sets `@timestamp` to ingest time if missing.  
```

Comment:
The component template references a default pipeline (`logs@default-pipeline`) which in turn sets `@timestamp` if missing.

<img width="539" height="464" alt="image" src="https://github.com/user-attachments/assets/23f04465-aff4-4535-bd6b-b1efc2d73a79" />